packages/components/psammead-paragraph/src/index.stories.jsx

```diff
@@ -5,9 +5,18 @@ import { storiesOf } from '@storybook/react';
 import { withKnobs } from '@storybook/addon-knobs';
 import InlineLink from '@bbc/psammead-inline-link';
 import { withServicesKnob } from '@bbc/psammead-storybook-helpers';
+import { create } from '@storybook/theming';
 import notes from '../README.md';
 import Paragraph from './index';
 
+const theme = create({
+  base: 'dark',
+  brandTitle: 'BBC Psammead',
+  brandUrl: 'https://github.com/bbc/psammead',
+  brandImage:
+    'https://user-images.githubusercontent.com/11341355/54079666-af202780-42d8-11e9-9108-e47ea27fddc5.png',
+});
```

Comment:
This will be replaced with this once it is merged: https://github.com/bbc/psammead/pull/3586